Packs/QualysVulnerabilityManagement/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_V2_README.md

```diff
@@ -0,0 +1,48 @@
+Manages vulnerabilities using the latest Qualys report.
+
+This playbook runs as a job, and by default creates incidents of type "Vulnerability" based on assets and vulnerabilities.
+The incidents are created from the latest version of the report determined by the report timestamp.
+
+You can define the minimum severity `minSeverity` that incidents are created for.
+Duplicate incidents are not created for the same asset ID and QID.
+
+This playbook is a part of a series of playbooks for Qualys vulnerability management and remediation.
+For this series of playbooks to run successfully, create a Job and do the following:
+1. Assign this playbook to the Job
+2. Enter the Qualys XML report name into the "Details" field
+3. Associate the "Vulnerability" type incident to the `Vulnerability Handling - Qualys` playbook.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* Qualys
+* Builtin
+
+### Scripts
+* QualysCreateIncidentFromReport
+* Set
+
+### Commands
+* qualys-report-fetch
+* closeInvestigation
+* qualys-report-list
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Source** | **Required** |
+| --- | --- | --- | --- | --- |
+| QualysReportTitle | The report title as it appears in Qualys. Has to be in XML format.  | details | incident | Required |
+| MinSeverity | The minimum Qualys severity (1 -5) to create incidents for. | 3 | - | Required |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Vulnerability_Management_Qualys_(Job)](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Vulnerability_Management_Qualys_(Job).png)
```

Comment:
retake the image while the Qualys instance is configured so the tasks will not have the `missing automation` icon

Reply:
Yes sure, I already took it but where should I upload it to?

Reply:
Added